src/stories/button.stories.tsx

```diff
@@ -127,16 +127,16 @@ stories.add("Circular", () => (
   <Button variant="primary" circular icon="heart" onClick={actionLog} />
 ));
 
-stories.add("Ripple", () => (
-  <>
-    <Button variant="primary" ripple>
-      Ripple
-    </Button>
-    &nbsp;
-    <Button variant="secondary" ripple>
-      Ripple
-    </Button>
-  </>
-));
+// stories.add("Ripple", () => (
```

Comment:
An example of ripple usage with button should be added and unnecessary comments should be deleted

```
<Ripple>
    <Button />
</Ripple>
```